README.md

```diff
@@ -70,14 +70,15 @@ modelerfour:
   group-parameters: true
   # Enable older inconsistent behavior that an enum with a single value would become a constant by default.
   seal-single-value-enum-by-default: true
+  lenient-model-deduplication: true

```

Comment:
I don't think we should enable this by default. This should be enabled in a case by case since it works around buggy swaggers and slows down the generation dramatically